source/guide/3.2/plugin-softlayer.md

```diff
@@ -0,0 +1,499 @@
+    * `location` The short name or id of the data center in which the VS should reside.
+    * `domain` The domain to use for the new server.
+      * for more information see [Resource Naming Convention](#resource-naming-convention)
+    * `ram` The item id of the desired server's RAM, e.g. item id 864 for 8 GB.
+    * `cpu` The item id of the desired server's CPU, e.g. item id 859 for 4 X 2.0 GHz Cores.
+    * `disk` The item id of the desired server's first disk, e.g. item id 1178 for 25 GB (SAN).
+
+  * **Optional Properties:**
+    * `api_config` 
+      * A dictionary containing the authentication information for connecting to the SoftLayer API:
+        - username: a SoftLayer username 
+        - api-key: a user-specific API Key
+        - endpoint_url: a softLayer endpoint URL of choice
+        
+        for more information see [SoftLayer-API-Overview](http://sldn.softlayer.com/article/SoftLayer-API-Overview)
+      * An empty dictionary by default - will be taken from other resources if not specified - see [SoftLayer authentication](#softlayer-authentication))  
+    * `hostname` The hostname to use for the new server, e.g. 'my-hostname'
+      * An empty string by default - will be generated automatically, see [Resource Naming Convention](#resource-naming-convention)
+    * `os` The item id of the operating system to use, e.g. item id 1857 for Windows Server 2008 R2 Standard Edition (64bit)
+    * `image_template_global_id` An image template global id to load the server with. 
+      * If an image is used, `os` must not be specified.
+    * `image_template_id` An image template id to load the server with. 
+      * If an image is used, `os` must not be specified.
+    * `quantity` The amount of servers to order
+      * default: 1
+    * `use_hourly_pricing` Flag to indicate whether this server should be billed hourly (default) or monthly
+      * default: true
+    * `private_network_only` Flag to indicate whether the computing instance only has access to the private network
+      * default: false
+    * `port_speed` The item id of the port speed
+      * default: 187 – the item id of 10 Mbps Public & Private Network Uplinks
+    * `private_vlan` The internal identifier of the private VLAN.
+      * By default SoftLayer will assign the private VLAN.
+    * `public_vlan` The internal identifier of the public VLAN. Cannot be declared if `private_network_only` flag is set to true.
+      * By default SoftLayer will assign the public VLAN.
+    * `provision_scripts` A list of the URIs of the post-install scripts to run after creating the server
+      * Each URI should start with https
+    * `ssh_keys` A list of SSH keys to add to the root user
+    * `bandwidth` The item id of the amount of bandwidth for this server
+      * default: 439 – the item id of 0 GB bandwidth
+    * `pri_ip_addresses` The item id of Primary IP Addresses
+      * default: 15 – the item id of 1 IP Address
+    * `monitoring` The item id of the monitoring
```

Comment:
of the monitoring what? Add something more here. Could be something like 'softlayer monitoring implementation', or something else.